clients/src/test/java/org/apache/kafka/clients/consumer/KafkaConsumerTest.java

```diff
@@ -1248,6 +1248,7 @@ public void closeShouldBeIdempotent() {
         KafkaConsumer<byte[], byte[]> consumer = newConsumer();
         consumer.close();
         consumer.close();
+        consumer.close();
```

Comment:
What purpose does the additional `close` serve over here? Does it reproduce a race condition some how?